pybossa/view/projects.py

```diff
@@ -3571,7 +3571,7 @@ def sync_project(short_name):
         # Perform sync
         is_new_project = False
         project_syncer = ProjectSyncer(
-            default_sync_target, target_key)
+            default_sync_target, target_key, current_app.config.get('DEFAULT_SYNC_TARGET_PROXIES'))
```

Comment:
Looks good, the request will use no proxy if the value for `DEFAULT_SYNC_TARGET_PROXIES` is not present in the config.